lib/config/config.py

```diff
@@ -146,6 +146,9 @@ def read_config(cls, path):
         :return:
         """
 
+        if not os.path.exists(path):
+            raise EnodoInvalidConfigException(f'Given config file does not exist or cannot be read')
```

Comment:
Maybe include the `path` which is missing in the exception text? It is already a "formatting" string and might help to debug when this exception is raised

Reply:
Done